docs/command_line_tools/command_line_usage/safedata_server_top.txt

```diff
@@ -0,0 +1,24 @@
+cl_prompt $ safedata_server -h
+usage: safedata_server [-h] [-r RESOURCES] [-q]
+                       {post_metadata,update_gazetteer} ...
+
+Post updated information to a safedata server instance/
```

Comment:
Should this be a slash at the end?

Reply:
Huh - fixed a flake8 typo and forgot to update usage